src/views/components/no-photo-id-exit-find-another-way-form.njk

```diff
@@ -0,0 +1,48 @@
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
```

Comment:
I have not edited these files, they are duplicates of the previous page but with reference to the new content reference